frontend/src/layouts/NetworkDropdown.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <q-btn-dropdown
```

Comment:
This format of `<q-btn>` and `<q-list>` works, but you could also use our own `<base-select>` component which might be a little cleaner / less code here.